File: app/src/main/java/com/cjstudio/sosestrada/Motorista.java
package com.cjstudio.sosestrada;

public class Motorista {
    private String uid, nome, telefone, email, veiculo, placa, cor;

    public Motorista() {} // necessário para Firestore

    // Getters e Setters (gerar todos)
    public String getUid() { return uid; }
    public void setUid(String uid) { this.uid = uid; }
    public String getNome() { return nome; }
    public void setNome(String nome) { this.nome = nome; }
    public String getTelefone() { return telefone; }
    public void setTelefone(String telefone) { this.telefone = telefone; }
    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }
    public String getVeiculo() { return veiculo; }
    public void setVeiculo(String veiculo) { this.veiculo = veiculo; }
    public String getPlaca() { return placa; }
    public void setPlaca(String placa) { this.placa = placa; }
    public String getCor() { return cor; }
    public void setCor(String cor) { this.cor = cor; }
}

File: app/src/main/java/com/cjstudio/sosestrada/Motorista.kt
package com.cjstudio.sosestrada

// Documento motoristas/{uid}. Valores padrão = construtor vazio exigido pelo
// Firestore no toObject().
data class Motorista(
    var uid: String? = null,
    var nome: String? = null,
    var telefone: String? = null,
    var email: String? = null,
    var veiculo: String? = null,
    var placa: String? = null,
    var cor: String? = null,
    // JPEG pequeno em base64 (ver FotoUtil) — não há Firebase Storage.
    var foto: String? = null,
    // Bloqueado pelo admin: não entra, não edita e não pede socorro.
    var bloqueado: Boolean = false
)
